src/configs/latest.ts

```diff
@@ -16,6 +16,7 @@
  */
 
 export const rules = {
+    "cyclomatic-complexity": true,
```

Comment:
Don't want to enable this by default, because I'm not sure cyclomatic complexity is something everyone would want to check by default. Thoughts?